app/publisher/publisher.go

```diff
@@ -18,18 +19,43 @@ type Interface interface {
 }
 
 // Stdout implements publisher.Interface and sends to stdout
-type Stdout struct{}
+type Stdout struct{
+	ExcludeList []string
+}
+
+// CheckExclusionList checks the exclusion list for matches
+func CheckExclusionList(excludes []string, msg string) bool {
+	for _, value := range excludes {
+		if len(value) > 0 && !strings.HasPrefix(value, "#") {
+			match, err := regexp.MatchString(strings.ToLower(value), strings.ToLower(msg));
+			if err != nil {
+				log.Printf("[WARN] regexp.MatchString error: %s", err)
+			}
```

Comment:
return false right away will make following the logic easier

Reply:
Explicit sad path return added